### PR TITLE
[V5] use selenium pie

### PR DESCRIPTION
**Pull request by emilyrohrbough**
### Summary
update to use selenium/hub:3.14.0-helium.

### CODE REVIEW

**Comment by emilyrohrbough** [comment]
FYI: nightwatch is failing but I didn't debug since we are removing in this MVB

File: config/docker/local-docker-compose.yml [hunk 1]
```diff
@@ -2,27 +2,26 @@ version: '3.4'
...
   chrome:
-    image: 'selenium/node-chrome:3.11.0-californium'
+    image: selenium/node-chrome:3.14.0-helium
     volumes:
       - /dev/shm:/dev/shm
-    ports:
-      - '5555:5555'
     deploy:
-      replicas: 5
+      replicas: 1
```
Comment (mjhenkes) [inline config/docker/local-docker-compose.yml:20]: if replicas are set to 1 can we remove the deploy settings all together? (i assume 1 is the default)

File: config/wdio/wdio.conf.js [hunk 2]
```diff
@@ -24,11 +24,15 @@ const config = {
     path.join('test*', 'wdio', '**', '*-spec.js'),
   ],
   maxInstances: 1,
-  capabilities: [
-    {
-      browserName: 'chrome',
+  capabilities: [{
+    browserName: 'chrome',
+    'goog:chromeOptions': {
+      /** Run in headless mode since Chrome 69 cannot reach the tiny viewport size due to a omnibox size change
```
Comment (bjankord) [inline config/wdio/wdio.conf.js:29]: Nice, thanks for adding this comment.

**Review by bjankord — APPROVED** [review]

**Comment by emilyrohrbough** [comment]
@bjankord its nightwatch tests and we are removing nightwatch so it doesn't matter

**Review by yuderekyu — APPROVED** [review]

**Pull request merged**